docs/articles/getting-started/observability.md

```diff
@@ -0,0 +1,117 @@
+Bytewax is instrumented to offer observability of your dataflow.
+
+The default configuration simply logs anything at the log level `ERROR` to standard output.  
+You can control the log level with an environment variable, `BYTEWAX_LOG`.  
+If you want to see all the messages bytewax emits (it is really verbose), you can set it to "trace":
+
+```
+BYTEWAX_LOG="bytewax=trace" python dataflow.py
+```
+
+The `TRACE` level is the most verbose, because it also sends traces to an opentelemetry compatible backend,
```

Comment:
Nope, traces are always sent to the backend, what I wanted to note is that the `TRACE` level is super verbose, and is the one you want to use if you want proper tracing, since all the spans are at the trace level. I'll rephrase this